snuba/consumers/consumer.py

```diff
@@ -566,7 +567,9 @@ def process_message(
             ),
         )
     except Exception as err:
-        logger.error(err, exc_info=True)
+        with sentry_sdk.push_scope() as scope:
```

Comment:
This has another issue though. 
the tag will be set to the warning message, not to the error message that is raised as InvalidMessage

Reply:
Indented the raise InvalidMessage so it is in this block too.